kolibri/core/assets/src/heartbeat.js

```diff
@@ -153,6 +153,8 @@ export class HeartBeat {
         })
       );
     }
+    // Log the time at the start of the request for time diff setting.
+    const pollStart = new Date();
```

Comment:
suuuper minor but I've always found `new Date()` to be an obfuscated way of getting the current date and time. Could possibly standardize on `Date.now()`

Reply:
Except that `Date.now()` returns the time in milliseconds since epoch, whereas `new Date()` returns a date object. Given how I end up using them in the calculation below, `Date.now()` would work, but I wouldn't necessarily want to standardize on it.